docs/_includes/head.html

```diff
@@ -69,7 +59,12 @@
       };
       anchors.add("h1, h2, h3, h4, h5");
     });
+    // Function to handle anchor links
+    document.addEventListener('click', function (event) {
+      if (event.target.tagName === 'A' && event.target.getAttribute('href').startsWith('#')) {
```

Comment:
@goldfishdolphin is this match of "A" case insensitive? If not, we might need to account for "a", too.